.circleci/config.yml

```diff
@@ -91,7 +91,17 @@ jobs:
       - image: circleci/node:10.18
     environment:
       CODE_PATTERN: (/* eslint-disable)|(// eslint-disable)
-      OVERALL_REVIEW_COMMENT: ESLint is disabled - _please_ wait for a VSP review.
+      OVERALL_REVIEW_COMMENT: >
+        # ESLint is disabled
+
+        `vets-website` uses ESLint to help enforce code quality. In most
+        situations we would like for to remain enabled.
```

Comment:
Change second sentence to "In most situations we would like ESLint to remain enabled."

Reply:
Oops, thank you for the reminder!  I made a typo when I meant to remove the `for` instead of the `it`.